docs/tutorial/part-one/index.md

```diff
@@ -169,7 +171,7 @@ export default () => (
 
 ![New about page](05-about-page.png)
 
-Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `/about`.
+Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `http://localhost:8000/about`.
```

Comment:
```suggestion
Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `[http://localhost:8000/about](http://localhost:8000/about)`.
```

Reply:
sorry - local links which results in 404 if not running should be only in code blocks and not be links

see: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#mark-localhost-urls-as-code-strings

Reply:
My bad :-)